cms/auth/tests/test_views.py

```diff
@@ -26,6 +26,7 @@ def _prep_request(self, req):
         SessionMiddleware(lambda r: None).process_request(req)
         req.session.save()
 
+        # Jake comment 2
         # attach a real FallbackStorage
         storage = FallbackStorage(req)
         req._messages = storage  # pylint: disable=protected-access
```

Comment:
Addressed: https://github.com/ONSdigital/dis-wagtail/pull/163#discussion_r2179680514

Reply:
@sanjeevz3009 comment 1 and 2 have been addresses in a different PR. it is best to annotate the bits relevant for this PR only

Reply:
Sorry @zerolab. Forgive my forgetfulness, do you know what PR? As I am struggling to find it...

Reply:
hmm, I take it back. Having re-reviewed this, the closed and original PRs, comments 1 and 2 have not been addressed in a different PR. Sorry!

The question is... did you mean to address them here (asking as they do not seem to be addressed here either)?

Reply:
Yes, I was meaning to... Addressed now